### ISSUE TITLE
Can't access to ESXi host from mRemoteNG

### ISSUE THREAD

**Issue by m0hossa** [mRemoteNG/mRemoteNG#2556]
Hi

I am trying to add ESXi host to access in web using HTTPs but it does not do anything. The version I am using is 1.77.3 (Build 1784).

Any help would be greatly appreciated.

Thanks

**Comment by Kvarkas** [comment]
what version of ESXi host?

**Comment by Meph79** [comment]
> what version of ESXi host?

I'm not the one reporting this problem but i have the same issue so does my collegues.. 
This is what i have tested:
Working fine with ESXi 7.0.3c
Does not work with ESXi 8.0.1 and 8.0.2

Im running mRemoteNG V 1.76.20.24615

I think it is not working since the new UI was introduced by Vmware... But not sure.. 
When loading the loginpage does not show up only a blank screen (with correct color background)

**Comment by simonai1254** [comment]
Has anyone tried another (random) webpage to be added into mremoteNG?

**Comment by Meph79** [comment]
> Has anyone tried another (random) webpage to be added into mremoteNG?

Yes. I use for example iLO 5 webpage to manage ProLiant servers, that works fine. 

ESXi pages look like this:
ESX 7.0.3: 
![image](https://github.com/mRemoteNG/mRemoteNG/assets/75944769/2a1b5afc-b3f7-42af-b116-42cd56f66bdb)

ESX 8.0.2: 
![image](https://github.com/mRemoteNG/mRemoteNG/assets/75944769/76736597-b49b-43f9-9e7b-20348b7ef889)

**Comment by m0hossa** [comment]
How can I add another webpage?


Thanks and Best Regards

Mohammad Sazzad Hossain

**Comment by journeym** [comment]
Why are u ppl trying to integrate browser into console session manager
Is this a new form of masohizm?

**Comment by m0hossa** [comment]
> > Has anyone tried another (random) webpage to be added into mremoteNG?
> 
> Yes. I use for example iLO 5 webpage to manage ProLiant servers, that works fine.
> 
> ESXi pages look like this: ESX 7.0.3: ![image](https://private-user-images.githubusercontent.com/75944769/297798162-2a1b5afc-b3f7-42af-b116-42cd56f66bdb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4MjA3NjAsIm5iZiI6MTcwNjgyMDQ2MCwicGF0aCI6Ii83NTk0NDc2OS8yOTc3OTgxNjItMmExYjVhZmMtYjNmNy00MmFmLWIxMTYtNDJjZDU2ZjY2YmRiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAxVDIwNDc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPThiNThmY2RmZmE3MmI1NjM0YWE3MWY2OThmNDcxNDAxZDIzY2FjNDFmMWEwYmU4N2JmOGNiNmMwOGRlYjM3ZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.evczXCBjzDyvERoZ9ljWj7fUfRORB2F4vlK1wGItHhU)
> 
> ESX 8.0.2: ![image](https://private-user-images.githubusercontent.com/75944769/297798329-76736597-b49b-43f9-9e7b-20348b7ef889.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4MjA3NjAsIm5iZiI6MTcwNjgyMDQ2MCwicGF0aCI6Ii83NTk0NDc2OS8yOTc3OTgzMjktNzY3MzY1OTctYjQ5Yi00M2Y5LTllN2ItMjAzNDhiN2VmODg5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjAxVDIwNDc0MFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI2Y2M5OWE3MzFhODFkNWQ5YTdlNjNjMjNhODlkYjg1NGIwODZjYWRiZmUxMTJlNDNmMWU5NGJlYWQ1YTlkYTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.WqK92eEbqrsqqaDgSMyyHr62JIPF_LucDbwCsrv4wtQ)

Can you please share a screen shot of the config page you added for iLO? If it works for iLO it should work for iDRAC too.

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `6c110776` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** Can't access to ESXi host from mRemoteNG

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
